x-pack/test/security_solution_endpoint_api_int/headers.ts

```diff
@@ -0,0 +1,11 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+export const HEADERS = {
+  'kbn-xsrf': 'xxx',
+  'x-elastic-internal-origin': 'xxx',
```

Comment:
maybe use a value of `security_solution` instead of `xxx`

Reply:
00f1bc1be53df303ca642e2fe8236ac85c78dfd5